linux/registry.cpp

```diff
@@ -60,19 +60,11 @@
 #endif
 
 // Convert a string that represents a hex value into an int
-int hextoi(char *p) {
-  int value = 0;
-  while ((p) && (*p) && isalnum(*p)) {
-    *p = toupper(*p);
-    if ((*p >= '0') && (*p <= '9'))
-      value = (value * 16) + ((*p) - '0');
-    else if ((*p >= 'A') && (*p <= 'F'))
-      value = (value * 16) + ((*p) - 'A');
-    else
-      return value;
-
-    p++;
-  }
+int hextoi(char *hexstring) {
+  int value;
+  std::stringstream ss;
```

Comment:
Need `#include <sstream>` for `std::stringstream`.

Reply:
fixed